packages/ng/inline-message/inline-message.component.html

```diff
@@ -1 +1,11 @@
-<lu-icon [icon]="state" *ngIf="state && state !== 'default'" [alt]="state"></lu-icon>{{label}}
+@if (isStringPortalContent(label)) {
+<p class="inlineMessage-content">
+	<span *ngIf="state && state !== 'default'" class="lucca-icon inlineMessage-content-statusIcon" aria-hidden="true"></span>
+	{{label}}
```

Comment:
```suggestion
	<ng-container *luPortal="label"></ng-container>
```

Reply:
(For my culture) what's the difference?

Reply:
`*luPortal` handles the case where it's a ng-template gracefully 👍